parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -351,9 +351,6 @@ MASTER_CONTAINER_ADDONS_PLACEHOLDER
     sed -i "s|<img>|{{WrapAsParameter "kubernetesCcmImageSpec"}}|g" /etc/kubernetes/manifests/cloud-controller-manager.yaml
     sed -i "s|<config>|{{GetK8sRuntimeConfigKeyVals .OrchestratorProfile.KubernetesConfig.CloudControllerManagerConfig}}|g" /etc/kubernetes/manifests/cloud-controller-manager.yaml
 {{end}}
-{{if not EnablePodSecurityPolicy}}
```

Comment:
These sed lines are no longer needed, as the `apparmor_parser` references were removed when we implemented kubelet running on the host OS:

https://github.com/Azure/acs-engine/pull/2584/files#diff-42c513b221406469e43868cefdc5ad55L30